pkg/workflows/validate.go

```diff
@@ -0,0 +1,91 @@
+package workflows
```

Comment:
It's difficult to know without understanding what this `Validate` is, but I think we shouldn't add anything to the workflows package
This probably belong in a `validations` subpackage?

Reply:
Ah ok I figured that it would make sense to have this in `workflows` since we are using a runner & the structure of this could be similar to our other workflows that use a runner (create, delete, etc.) but I see your point. Makes sense to add it to its own subpackage